### DISCUSSION TITLE
how to generate doc tags

### DISCUSSION THREAD

**Discussion by oeyoews** [nvim-neorg/neorg#266] [General]
I noticed that the neorg plugin uses the norg format to write documents. Did it use other auxiliary tools to generate the tags? I'm curious about it

**Comment by mrossinek** [comment]
No, we do not yet have an automated tool to generate the tags. The initial file was created by me by hand and we have been manually updating this since then.

I have some plans for how to leverage the table of contents functionality provided by #159 in the future. But this will be limited to headings as things stand now.
